docs/spec.md

```diff
@@ -2720,79 +2720,9 @@ If not specified, all dimensions are assumed to be possibly expanding.
-
-* `padding_value = constant(0, element_type(lhs))`.
-* `padded_lhs = pad(lhs, padding_value, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations - 1)`.
-* `lhs_window_start = lhs_shape(0, output_spatial_index, 0) * lhs_window_strides`.
-* `lhs_window = slice(padded_lhs, lhs_window_start, lhs_window_start + lhs_window_dimensions, lhs_window_dilations)`.
-* `reversed_lhs_window = reverse(lhs_window, [input_spatial_dimensions[dim] for dim in range(size(window_reversal)) if window_reversal[dim] = true])`.
-  This feature appears to be unused, so in the future we are planning to remove
-  it ([#1181](https://github.com/openxla/stablehlo/issues/1181)).
-* `dot_product = dot_general(reversed_lhs_window, rhs,
-    lhs_batching_dimensions=[],
-    lhs_contracting_dimensions=input_spatial_dimensions + [input_feature_dimension],
-    rhs_batching_dimensions=[],
-    rhs_contracting_dimensions=kernel_spatial_dimensions + [kernel_input_feature_dimension])`.
-
-If `feature_group_count > 1`:
-
-* `lhses = split(lhs, feature_group_count, input_feature_dimension)`.
-* `rhses = split(rhs, feature_group_count, kernel_output_feature_dimension)`.
-* `results... = convolution(lhses..., rhses..., ..., feature_group_count=1, ...)`.
-* `result = concatenate(results, output_feature_dimension)`.
-
-If `batch_group_count > 1`:
-
-* `lhses = split(lhs, batch_group_count, input_batch_dimension)`.
-* `rhses = split(rhs, batch_group_count, kernel_output_feature_dimension)`.
-* `results... = convolution(lhses..., rhses..., ..., batch_group_count=1, ...)`.
-* `result = concatenate(results, output_feature_dimension)`.
-<!-- markdownlint-enable line-length -->
-
-For quantized types, performs `dequantize_op_quantize(
-    lambda lhs, rhs: convolution(lhs, rhs, d_padding, window_strides,
-        lhs_dilation, rhs_dilation, window_reversal, input_batch_dimension,
-        input_feature_dimension, input_spatial_dimensions,
-        kernel_input_feature_dimension, kernel_output_feature_dimension,
-        kernel_spatial_dimensions, output_batch_dimension,
-        output_feature_dimension, output_spatial_dimensions,
-        feature_group_count, batch_group_count, precision_config), lhs, rhs,
-        type(result))`.
-
-For hybrid quantized types, performs `hybrid_dequantize_then_op(
-    lambda lhs, rhs: convolution(lhs, rhs, d_padding, window_strides,
-        lhs_dilation, rhs_dilation, window_reversal, input_batch_dimension,
-        input_feature_dimension, input_spatial_dimensions,
-        kernel_input_feature_dimension, kernel_output_feature_dimension,
-        kernel_spatial_dimensions, output_batch_dimension,
-        output_feature_dimension, output_spatial_dimensions,
-        feature_group_count, batch_group_count, precision_config), lhs, rhs)`.
+This operation is functionally identical to
+[convolution](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#convolution)
+op, but the padding is specified dynamically via `d_padding`.
```

Comment:
Can you also address the comment at https://github.com/openxla/stablehlo/pull/2312#discussion_r1596933457?